server/athenian/api/models/precomputed/models.py

```diff
@@ -40,7 +40,7 @@ class GitHubDonePullRequestFacts(Base, UpdatedMixin):
 
     pr_node_id = Column(CHAR(32), primary_key=True)
     release_match = Column(Text(), primary_key=True)
-    format_version = Column(Integer(), primary_key=True, default=6, server_default="6")
```

Comment:
@vmarkovtsev you bumped these because of the new `reviews` field in the `PullRequestFacts` dataclass, right?

Reply:
Every time we change the facts, pdb has to be reset. The fundamental pitfall of precomputation :smile: